pkg/daemon/controller/controller.go

```diff
@@ -282,15 +282,16 @@ func (sc *ServiceController) Reconcile(_ context.Context, req ctrl.Request) (res
 	return ctrl.Result{}, nil
 }
 
-// reconcileEnvironmentVariables makes sure that the proxy variables exist as expected, or are safely rectified.
+// reconcileProxySettings makes sure that the proxy variables exist as expected, or are safely rectified.
```

Comment:
I am not sure about this change. WICD should be generic wrt environment variables. It shouldn't be proxy specific. These variables just so happen to be related to enabling proxy in the environment.

Reply:
I see that you are using this function to reconcile certs in the next commit, if so it can be named along the lines of
reconcileEnvVarsAndCerts or you could create another function that calls both of these.

Reply:
I'll rename the func and squash this commit with the next so it's clear how it'll be used